lib/presentation/getx/controllers/cart_controller.dart

```diff
@@ -0,0 +1,107 @@
+import 'dart:convert';
+
+import 'package:ecom_mvvm/data/models/product_model.dart';
+import 'package:get/get.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class CartController extends GetxController {
+  RxMap<ProductModel, int> cartItems = <ProductModel, int>{}.obs;
```

Comment:
Explain why not created a RxList ?

Reply:
RxMap provides a unique key value for each order or cart item Here We don't have a cart model and had to count the quantity. So I preferred using RxMap over RxList.